packages/compiler-cli/src/transformers/patch_alias_reference_resolution.ts

```diff
@@ -60,32 +63,47 @@ interface EmitResolver {
  * `emitDecoratorMetadata` flag is enabled. TypeScript basically surfaces the same problem and
  * solves it conceptually the same way, but obviously doesn't need to access an `@internal` API.
  *
+ * The set that is returned by this function is meant to be filled with import declaration nodes
+ * that have been referenced in a value-position by the transform, such the the installed patch can
+ * ensure that those import declarations are not elided.
+ *
  * See below. Note that this uses sourcegraph as the TypeScript checker file doesn't display on
  * Github.
  * https://sourcegraph.com/github.com/microsoft/TypeScript@3eaa7c65f6f076a08a5f7f1946fd0df7c7430259/-/blob/src/compiler/checker.ts#L31219-31257
  */
-export function patchAliasReferenceResolutionOrDie(
-    context: ts.TransformationContext, referencedAliases: Set<ts.Declaration>): void {
+export function loadAliasReferenceResolutionPatchOrDie(context: ts.TransformationContext):
```

Comment:
For consistency:

```suggestion
export function loadIsReferencedAliasDeclarationPatchOrDie(context: ts.TransformationContext):
```

And I would probably drop the `OrDie` postfix as it is just an implementation aspect.